src/app/helptext/system/cloudcredentials.ts

```diff
@@ -307,15 +307,9 @@ export const helptext_system_cloudcredentials = {
     tooltip: T("Password for the SSH Username account.")
   },
 
-  key_file_sftp: {
-    placeholder: T("PEM-encoded private key file path"),
-    tooltip: T(
-      'Path to an unencrypted <a\
- href="https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail"\
- target="_blank">PEM-encoded</a> private key file.\
- Example: <i>/home/$USER/.ssh/id_rsa</i>. Leave blank\
- to use <i>ssh-agent</i>.'
-    )
+  private_key_sftp: {
+    placeholder: T("Private Key ID"),
+    tooltip: T('')
```

Comment:
Rough draft help text:
"Import the private key from an existing SSH keypair. Choose the name of the SSH keypair to use for this connection."